### ISSUE TITLE
Bug: the quickui menu become colorless after using set background cmd

### ISSUE THREAD

**Issue by nash900** [skywind3000/vim-quickui#8]
After run ':set bg=dark', this issue will happen. 
![menu](https://user-images.githubusercontent.com/8497245/72678685-4d793e00-3ae3-11ea-8664-d0245474fd59.png)

BTW, thanks a lot for adding this wonderful feature.

**Comment by nash900** [comment]
I found if run ":call QuickThemeChange('')" after bg cmd can get the menu color back.

**Comment by skywind3000** [comment]
Yes, maybe changing background will break highlight links. Still need to look into.

**Comment by roachsinai** [comment]
even `:call QuickThemeChange('default')` not works now days.

**Comment by saidelmark** [comment]
I like this "theme", to be honest. Two things that bugs me are that
1. I can't see which line is selected, because the highlighting is broken.
2. I use tabline provided by airline plugin, and it breaks too.
before re-sourcing my vimrc:
![image](https://user-images.githubusercontent.com/9284083/98846861-99c1b000-2460-11eb-8201-b1f88f3effb6.png)
after (looks cool though, maybe will customize the theme):
![image](https://user-images.githubusercontent.com/9284083/98846966-bf4eb980-2460-11eb-8161-315702fe93a0.png)
